app/panel/components/Help.jsx

```diff
@@ -23,9 +26,11 @@ const Help = () => (
 		<div className="row">
 			<div className="small-12 columns">
 				<h1>{t('panel_help_panel_header')}</h1>
-				<div className="support-section">
-					<a href="#" onClick={openHubPage}>{t('panel_help_setup')}</a>
-				</div>
+				{ BROWSER_INFO.name !== 'ghostery_desktop' && (
```

Comment:
So we completely disable access to the hub page on the browser? Would it not be useful for some users who may want to access it later?

Reply:
We would need to adjust opt-in default settings for the hub but it's doable.

Reply:
Fixed